plugins/gradle/src/index.ts

```diff
@@ -39,12 +39,16 @@ export interface IGradleProperties {
  * Builds properties object from gradle properties command
  *
  * @param gradleCommand - base gradle command
+ * @param gradleOptions - options to pass to gradle command
  * @returns properties wrapped in a promise
  */
 export async function getProperties(
-  gradleCommand: string
+  gradleCommand: string,
+  gradleOptions: string[]
 ): Promise<IGradleProperties> {
-  const properties = (await execPromise(gradleCommand, ["properties", "-q"]))
+  const properties = (
```

Comment:
Not sure it matters, but any reason we don't do "-q" before we do "properties" ...  Looks better in that order...